### DISCUSSION TITLE
5.0.0alpha3

### DISCUSSION THREAD

**Discussion by niden** [phalcon/cphalcon#15565] [General]
All notable changes to this version has been documented in the [CHANGELOG-5.0.md](https://github.com/phalcon/cphalcon/blob/5.0.0alpha3/CHANGELOG-5.0.md) file.

Supported PHP versions: 7.4, 8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/phalcon/cphalcon/releases/tag/5.0.0alpha3'>5.0.0alpha3</a>.</em>

**Comment by quasipickle** [comment]
Broken link.

https://github.com/phalcon/cphalcon/blob/5.0.0alpha3/CHANGELOG-5.0.md

**Comment by Arhell** [comment]
Fix

**Comment by fogelfish** [comment]
Not able to install this version of Phalcon in Windows:

> Windows Server 2019
> PHP 8.0.10
> php_psr-1.1.0-8.0-ts-vs16-x64
> phalcon_x64_vs16_php8.0 (5.0.0alpha3)

php_psr.dll and php_phalcon.dll are sitting in the php ext directory; the extensions are declared in php.ini (I put php_psr before php_phalcon and tried declaring both before or after other extensions, but I don't see either PSR or Phalcon in phpinfo().  Am I missing something?  Thank you for possible assistance.

**Reply by Jeckerson** [reply]
Please show your php.ini and how you declared extensions? Also what kind of error appears?

**Reply by fogelfish** [reply]
My mistake!  I could see the extensions in `php -i` but not in the browser because I neglected to add the **PHPIniDir** declaration in httpd.conf.  The bottom of my httpd.conf file now looks like this and I can see the extensions in the phpinfo() output.

```
# PHP 8.0
PHPIniDir "C:\PHP\php-8.0.10-Win32-vs16-x64"
LoadModule php_module "C:\PHP\php-8.0.10-Win32-vs16-x64\php8apache2_4.dll"
<FilesMatch \.php$>
    SetHandler application/x-httpd-php
</FilesMatch>
```
